src/_sass/_utilities.scss

```diff
@@ -17,18 +17,35 @@ $mm-dark-tan: rgb(246, 243, 237);
   background-color: $gray-lightest;
 }
 
-.text-red {
-  color: red;
-}
-
 .bg-dark-blue {
   background-color: $dark-blue;
 }
 
+// Text
+
+.text-red {
+  color: red;
+}
+
 .text-gray-4 {
   color: $gray-4;
 }
 
+.fs-xs-5 {
+  font-size: calc(20rem / 16);
+
+  @include media-breakpoint-down(md) {
+    font-size: calc(24rem / 16);
+  }
+}
```

Comment:
Would it make more sense to have the rules for this class wrapped in a media query specifically for the `xs` size? 

Then we don't have to undo/override anything else in this class, and it won't interact with any other sizing rules for larger screens elsewhere.

In other words, this class "says" it is for `xs-5`, but then it also has rules for `md-4`, which seems unnecessary?

Reply:
Resolved here https://github.com/cal-itp/mobility-marketplace/pull/434/commits/b191d9a929c41e74ca75f1be0467535880c9aa40